src/js/construct-element.js

```diff
@@ -16,21 +16,7 @@ export default {
 			alertMessageEl.classList.add(`${opts.statusClass}`);
 		}
 
-		let primaryActionHTML;
-		if (opts.actions.primary.text) {
-			primaryActionHTML = `<a href="${opts.actions.primary.url}" class="${opts.messageClass}__action--primary">${opts.actions.primary.text}</a>`;
-		}
-
-		let secondaryActionHTML;
-		if (opts.actions.secondary.text) {
-			secondaryActionHTML = `<a href="${opts.actions.secondary.url}" class="${opts.messageClass}__action--secondary">${opts.actions.secondary.text}</a>`;
-		}
-
-		let actions = `<div class="${opts.messageClass}__actions">
-			${primaryActionHTML}
-			${secondaryActionHTML}
-			</div>
-		`;
+		let actions = module.exports.actions(opts);
```

Comment:
It's not necessary but if construct-element was an ES6 class we could use static methods and avoid node specific code :D